WcaOnRails/app/views/regulations/translations/index.html.erb

```diff
@@ -1,88 +1,143 @@
-    </ul></li>
-    <li><%= link_to "Portuguese, Brazilian (Português do Brasil) - 2018-01-01", "./portuguese-brazilian/" %>
-    <ul>
-      <li>By <%= link_to "João Gabriel de Aguiar Milani", person_path("2011MILA02") %>, <%= link_to "Marlon de Vasconcelos Marques", person_path("2014MARQ02") %>, and <%= link_to "Pedro Santos Guimarães", person_path("2007GUIM01") %>.</li>
-    </ul></li>
-    <li><%= link_to "Portuguese, European (Português Europeu) - 2015-07-01", "./portuguese-european/" %>
-    <ul>
-      <li>By António Gomes.</li>
-    </ul></li>
-    <li><%= link_to "Russian - 2018-01-01", "./russian/" %>
-    <ul>
-      <li>By <%= link_to "Anton Rostovikov", person_path("2009ROST01") %>, <%= link_to "Oleg Gritsenko", person_path("2011GRIT01") %>, and <%= link_to "Nikolay Masson", person_path("2011MASS01") %> (with the help of <%= link_to "Oksana Ruzaeva", person_path("2010RUZA01") %>, <%= link_to "Anton Goryachikh", person_path("2009GORY01") %> and <%= link_to "Dmitry Aniskin", person_path("2011ANIS01") %>).</li>
-    </ul></li>
-    <li><%= link_to "Spanish, American (Español Americano) - 2015-07-01", "./spanish-american/" %>
-    <ul>
-      <li>By Daniela Maldonado Hernández.</li>
-    </ul></li>
-    <li><%= link_to "Spanish, European (Español de España) - 2015-07-01", "./spanish-european/" %>
-    <ul>
-      <li>By <%= link_to "Luis J. Iáñez", person_path("2009PARE02") %>, <%= link_to "Carlos Méndez García-Barroso", person_path("2010GARC02") %>, <%= link_to "Raúl Morales", person_path("2013MORA02") %>, and <%= link_to "Federico Sánchez Motellón", person_path("2009SANC01") %>.</li>
-    </ul></li>
-    <li><%= link_to "Vietnamese - 2018-01-01", "./vietnamese/" %>
-    <ul>
-      <li>By <%= link_to "Trung Vương Thiện", person_path("2014TRUN01") %>.</li>
-    </ul></li>
-  </ul>
-  <h2>Translating the Regulations</h2>
-  <p>If you're interested in adding/updating a translation, please view <%= link_to "the instructions on the GitHub repository for translations", "https://github.com/thewca/wca-regulations-translations#translation-instructions" %>.</p>
+  <p><%= t("regulations_translations.paragraph1") %></p>
+  <p><%= t("regulations_translations.paragraph2") %></p>
+
+  <h2><%= t("regulations_translations.translations") %></h2>
+
+  <% current_translations = [
+        {
+          version: "2019-05-01",
+          language: "简体中文",
+          language_english: "Chinese",
+          url: "./chinese",
+        },
+        {
+          version: "2019-05-01",
+          language: "日本語",
+          language_english: "Japanese",
+          url: "./japanese",
+        },
+      ] %>
```

Comment:
The indentation seems a bit of, what about:
```erb
<% current_translations = [
  {
    version: "2019-05-01",
    language: "简体中文",
    language_english: "Chinese",
    url: "./chinese",
  },
  {
    version: "2019-05-01",
    language: "日本語",
    language_english: "Japanese",
    url: "./japanese",
  },
] %>
```

Reply:
Ups, right!